modules/platforms/cpp/core-test/CMakeLists.txt

```diff
@@ -77,5 +77,7 @@ set(TEST_TARGET IgniteCoreTest)
 
 add_test(NAME ${TEST_TARGET} COMMAND ${TARGET} --catch_system_errors=no --log_level=all)
 
-set_tests_properties(${TEST_TARGET} PROPERTIES ENVIRONMENT
-        IGNITE_NATIVE_TEST_CPP_CONFIG_PATH=${PROJECT_SOURCE_DIR}/config)
+string(CONCAT ENV_VARIABLES "ASAN_OPTIONS=handle_segv=0:detect_leaks=0;UBSAN_OPTIONS=print_stacktrace=1;"
+                            "IGNITE_NATIVE_TEST_CPP_CONFIG_PATH=${PROJECT_SOURCE_DIR}/config")
+
+set_tests_properties(${TEST_TARGET} PROPERTIES ENVIRONMENT "${ENV_VARIABLES}")
```

Comment:
Should not we add these options only if sanitizers are enabled?

Reply:
Added check if properties are set and if not, set default